org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/NullReferenceTest.java

```diff
@@ -18568,4 +18568,130 @@ void m(String packageName, IPS[] packages) {
 			"""
 		});
 }
+// https://github.com/eclipse-jdt/eclipse.jdt.core/issues/1461
+public void testGH1461() {
+	if (this.complianceLevel == 0) {
+		buildAllCompliancesTestSuite(testClass());
+		this.complianceLevel = ClassFileConstants.JDK15;
+	}
```

Comment:
I missed this block during first review. What is the purpose of this block?

Reply:
...to run the test in Eclipse.

Reply:
This shouldn't be needed. Are you running tests as *JUnit3*?

In case you want to run only one or a few specific tests, you can temporarily edit the `static` block at the top of the file like this:
```
static {
	TESTS_NAMES = new String[] { "testGH1461" };
}
```

I know it's a bit unusual, but it works :)

Reply:
> I missed this block during first review. What is the purpose of this block?

I have removed this block.